src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -13,7 +13,7 @@ import {
   Button,
 } from 'components';
 import {ResourceListContext} from '../../../utilities/resource-list';
-import ResourceItem from '../ResourceItem';
+import {ResourceItem} from '../ResourceItem';
```

Comment:
🤞 this is the only case of importing it... is there a chance external users would have been depending on the default export here?

Reply:
If they are then they're doing something unstable and unsupported :) Importing from `@shopify/polaris` is the public interface, all other JS is considered private and we're free to change it whenever.

They should be doing `import {ResourceItem} from '@shopify/polaris'`; which is the public export of this component.